### ISSUE TITLE
jQuery method .data() not working with TurnJS

### ISSUE THREAD

**Issue by lightmar** [blasten/turn.js#176]
Hi,

I have found out that everywhere I use the the jQuery method $(element).data("owner") there is a problem.
I use $(element).data("owner") = obj in order to assign data to a DOM element, and after that I am retrieving that data by using obj = $(element).data("owner"). Every place I try to retrieve a reference like that, the $(element).data("owner") is undefined.

Please help.

**Comment by blasten** [comment]
Yes, you right as well as events. This happens because turn.js removes some pages from the DOM tree when these are not necessary to the reader. The remove operation uses jQuery method `remove`, but it also removes data and event listeners of everything inside. The next version will fix this problem by replacing `remove` by `detach`.

To solve the problem right now, please go to the line 680 and replace:

``` javascript
data.pages[page].remove();
```

By

``` javascript
data.pages[page].detach();
```

And line 685 

``` javascript
data.pageObjs[page].remove();
```

by

``` javascript
data.pageObjs[page].detach();
```

To create the minimized version, you need to install Google closure (https://developers.google.com/closure) and run the script compress.sh

**Comment by lightmar** [comment]
Thanks, it worked.
I am still having some issues, I am figuring out where they come from.
(by the way, the changes weren't on line 680 and 685, they were in other lines)

**Comment by blasten** [comment]
@lightmar Sorry, I probably looked at a different version. Are you using 4.0.9?

**Comment by lightmar** [comment]
I am using Turn.js 3rd release.
